garmin.py

```diff
@@ -18,6 +18,10 @@ class LoginFailed(Exception):
     pass
 
 
+class APIException(Exception:
```

Comment:
There is a missing ')'

Reply:
Dang, not sure how I managed to miss that - thanks, fixed!